message msg_game{
	required int32 id = 1;
	required string tipo = 2;
	optional int32 x = 3;
	optional int32 y = 4;
	optional string nombre = 5;
	optional string imagen = 6;
	optional int ancho_base = 7;
	optional int alto_base = 8;
	optional int fps = 9;
	optional int delay = 10;
	optional int total_frames_line = 11;
	optional int pixels_dimension = 12;
}